src/components/Navbar.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-
+import accIcon from '../assets/images/profile.png';
+import "./styles/navBar.css"
 const Navbar = () => (
-  <nav>
-    <p>Book Store</p>
-    <ul>
-      <li><Link to="/">Books</Link></li>
-      <li><Link to="/categories">Categories</Link></li>
-    </ul>
+  <nav className='navBar'>
+    <div className="navContainer">
+      <p className='logo'>Bookstore CMS</p>
+      <ul className='navLinks'>
+        <li><Link to="/">Books</Link></li>
+        <li><Link to="/categories">Categories</Link></li>
```

Comment:
- [ ] The navbar menu doesn't match the Figma design. Kindly match the font color. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![your1](https://user-images.githubusercontent.com/70483835/222631020-93a21a12-acf6-4894-aa73-2d226cfc321a.png) | ![figma1](https://user-images.githubusercontent.com/70483835/222631478-59160a74-3cb8-49cd-b231-8551f9821674.png)